docs/components/slideout.md

```diff
@@ -77,6 +282,12 @@ There is one default slot which is used to place content into the slideout panel
 </KSlideout>
 ```
 
+- `badgeContent` - used to customize `badge` inside the header
```

Comment:
suggestion to use three slots: `title-before`, `title`, and `title-after` as discussed above